### DISCUSSION TITLE
E2112 IKEA Vindstyrka air quality and humidity sensor can't add it to Zigbee2MQTT

### DISCUSSION THREAD

**Discussion by bybyle** [Koenkk/zigbee2mqtt#16883] [Devices]
Hello,

I am not sure how to add the new sensor from Ikea to zigbee2mqtt. Even after the update, I cant discover it automatically and I have no clue how to manually discover it. 

Could you guys help me out?

Thanks !

**Comment by diplix** [comment]
you have to factory reset the device, four or five (or six?) short presses on the small top button.
